test/executor.test.js

```diff
@@ -906,6 +906,62 @@ describe('executor', function() {
     });
   });
 
+  describe('when booting with lazy connect', function() {
+    var sampleInstruction = someInstructions({
+      dataSources: {
+        lazyConnector: {
+          connector: 'testLazyConnect',
+          name: 'lazyConnector',
+        },
+      },
+    });
+    var connectTriggered = true;
```

Comment:
Nitpick: the value is always reset in `beforeEach` hook below, there is no need to assign a value here.

```js
var connectTriggered;
beforeEach(function() {
  connectTriggered = true;
  // ...
});
```